typescript/src/customerInfo.ts

```diff
@@ -311,3 +317,16 @@ export interface PurchasesSubscriptionInfo {
      */
     readonly willRenew: boolean;
 }
+
+/**
+ * Describes information about a virtual currency in the app.
+ * 
+ * Use this class to access information about a virtual currency, such as its current balance.
+ * @public
+ */
+export interface PurchasesVirtualCurrencyInfo {
+    /**
+     * The current balance of the virtual currency.
+     */
+    readonly balance: bigint;
```

Comment:
Huh interesting... I've been using `number` for the types of these... I guess `bigint` might be more specific but I wonder if we should keep `number` for simplicity + consistency...

Reply:
yeah I think we might be limited by int64 size elsewhere so using bigint here might not bring any benefit

Reply:
@tonidero We had a conversation in the virtual currencies Slack channel and decided to limit the max VC balance size to a signed 32-bit integer, so it'll definitely be safe to change this to a `number`. I'll make the change 😄

Reply:
Changed in 6f4849002637793a4982251c34237b525a3c09cb :D